jcontroller.php

```diff
@@ -173,6 +178,34 @@ protected function refRequest($param)
         return $this->refRequest[$param];
     }
 
+    /**
+     * Filters access to an action based on the controller methods.
+     *  
+     * If access is granted, it proceeds to display the action.
+     * If access is not granted, it displays an error code (401 - Access Forbidden).
+     * 
+     * @return void
+     */
+    public function filterAccessControl()
```

Comment:
I dont think this will prevent anything. Please make sure to call this from the `constructor`. Also why is this method public?

Reply:
I made it public because I initially used it from jRouter itself. As I have called this function from the constructor itself, now having the function as protected still works.

Reply:
make it private please